src/style.css

```diff
@@ -0,0 +1,117 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+.form {
+  position: absolute;
+  top: 20px;
+  left: 10px;
+  right: 10px;
+}
+
+.form--theme-small {
+  top: 110px;
+}
+
+.form__input {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 70px;
+  padding: 0 0 0 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-weight: 300;
+  font-family: "Avenir", sans-serif;
```

Comment:
`font-family: "Avenir", sans-serif;` - just set it to the `html `selector and inherit where needed